src/models/StyledNativeComponent.js

```diff
@@ -95,8 +96,23 @@ export default (constructWithOptions: Function) => {
       }
     }
 
+    setNativeProps(nativeProps: Object) {
+      this.root.setNativeProps(nativeProps)
+    }
+
+    generateRef() {
```

Comment:
You don't need to generate a new method on every call, but can instead reuse the same method for the callback over and over again :wink: 🎐